diff --git a/dist/index.mjs b/dist/index.mjs
index d63bec6afafabf835722bbbbf98354936a0040e1..450af9aacb462b1802d1db7a16d0dc47af70fbf8 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -2032,6 +2032,7 @@ function ClientManifestPlugin(nuxt) {
 	let clientEntry;
 	let key;
 	let disableCssCodeSplit;
+	let manifestProcessed = false;
 	return {
 		name: "nuxt:client-manifest",
 		applyToEnvironment: (environment) => environment.name === "ssr",
@@ -2041,6 +2042,8 @@ function ClientManifestPlugin(nuxt) {
 			disableCssCodeSplit = config.build?.cssCodeSplit === false;
 		},
 		async closeBundle() {
+			if (manifestProcessed) return;
+			manifestProcessed = true;
 			const devClientManifest = {
 				"@vite/client": {
 					isEntry: true,
